client/src/entrypoints/admin/telepath/blocks.js

```diff
@@ -114,6 +141,20 @@ class StructBlock {
     }
   }
 
+  setError(errorList) {
+    if (errorList.length !== 1) {
+      return;
+    }
+    const error = errorList[0];
+
+    // eslint-disable-next-line no-restricted-syntax
+    for (const blockName in error.blockErrors) {
+      if (error.blockErrors.hasOwnProperty(blockName)) {
+        this.childBlocks[blockName].setError(error.blockErrors[blockName]);
+      }
+    }
```

Comment:
I [found elsewhere](https://github.com/wagtail/wagtail/blob/53678dd0d9fa1167698c3fd34c745af5eb01f000/client/src/utils/telepath.js#L108-L110) that a `for ... of` loop is a nicer way of looping over a dict...
```
    for (const [blockName, blockError] of Object.entries(error.blockErrors)) {
      this.childBlocks[blockName].setError(blockError);
    }
```